src/main/resources/reference.conf

```diff
@@ -11,7 +11,7 @@ dataproc {
   networkTag = "leonardo"
   defaultScopes = ["https://www.googleapis.com/auth/userinfo.email", "https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/bigquery", "https://www.googleapis.com/auth/source.read_only"]
   # TODO: temp 2019-06-07
-  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:0b28747"
+  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:latest"
```

Comment:
we don't want to always use latest tag because that means we can potentially be using unstable welder in prod that's not tested...Before merging this PR, we should update this with an appropriate hash (we don't have it yet, but hopefully, once https://github.com/DataBiosphere/welder/pull/15 is merged, we'll have a good tag to use

Reply:
you can use `us.gcr.io/broad-dsp-gcr-public/welder-server:fd981fe`